compiler.js

```diff
@@ -1,3 +1,68 @@
+let examples = new Map();
+
+examples["helloWorld"] = `// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+contract MyContract {
+    function helloWorld() public pure returns (string memory) {
+        return "Hello, World!";
+    }
+}`;
+
+examples["erc20"] = `// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract ERC20 {
+    string public constant name = "ERC20";
+    string public constant symbol = "ERC";
+    uint8 public constant decimals = 18;
+
+    mapping(address => uint256) balances;
+    mapping(address => mapping (address => uint256)) allowed;
```

Comment:
Maybe we should remove the allowance feature to make the example simpler?

Reply:
Yes, I will remove it